src/main/java/mate/academy/springboot/swagger/util/SortUtill.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortUtill {
+    public static List<Sort.Order> createSortOrders(String sortBy) {
```

Comment:
```suggestion
    public List<Sort.Order> createSortOrders(String sortBy) {
```
don't use static if it's possible, it's a bad practice